lib/Input.js

```diff
@@ -30,23 +40,25 @@ class Keyboard {
    * @param {string} key
    * @param {{text: string}=} options
    */
-  async down(key, options = {text: undefined}) {
-    let { text } = options;
-    // If the key is a single character, and no modifiers are pressed except shift, a keypress should be generated by default.
-    if (text === undefined)
-      text = (key.length === 1 && !(this._modifiers & ~8)) ? key : '';
-    const autoRepeat = this._pressedKeys.has(key);
-    this._pressedKeys.add(key);
-    this._modifiers |= this._modifierBit(key);
+  async down(key, options = { text: undefined }) {
+    const description = this._keyDescriptionForString(key);
```

Comment:
this should throw if there's no description for the key

Reply:
Done.